pkg/apis/agones/v1/gameserver.go

```diff
@@ -161,6 +178,8 @@ type GameServerSpec struct {
 	// (Alpha, PlayerTracking feature flag) Players provides the configuration for player tracking features.
 	// +optional
 	Players *PlayersSpec `json:"players,omitempty"`
+	// (Alpha, SafeToEvict feature flag) Eviction specifies the eviction tolerance of the GameServer.
```

Comment:
We seem pretty inconsistent here. Health, Scheduling, SdkServer can all be left unspecified as well. I added it because I agree with you, but I wonder if we should look at this more carefully.